include/aos/common/tools/fs.hpp

```diff
@@ -358,8 +359,46 @@ class FS {
      */
     static Error ReadFileToString(const String& fileName, String& text)
     {
-        (void)fileName;
-        (void)text;
+        auto fd = open(fileName.CStr(), O_RDONLY);
+        if (fd != 0) {
+            return AOS_ERROR_WRAP(ErrorEnum::eFailed);
```

Comment:
Do not use AOS_ERROR_WRAP in lib.
aos::Error handles errno, so you should return: `return errno;` for all the io ops.

Reply:
fixed